packages/clerk-js/src/ui/elements/IdentityPreview.tsx

```diff
@@ -30,12 +30,7 @@ export const IdentityPreview = (props: IdentityPreviewProps) => {
   );
 
   if (!refs.current.identifier) {
-    return (
-      <Container {...rest}>
-        <Authenticator />
```

Comment:
Why are we removing this?

Reply:
I was following the designs and that part has been removed.

Reply:
Is that just the icon ? Can you still sign in w./ auth app normally?

Reply:
yeah it's only the icon and the text, everything else works normally!